pages/[filename].tsx

```diff
@@ -21,14 +21,14 @@ export default function HomePage(
       <Layout>
         <Blocks prefix="PageBeforeBody" blocks={data.page.beforeBody} />
         <Container className={"flex-1 pt-4"}>
-          <div className="gap-4 md:grid md:grid-cols-5 lg:grid-cols-3">
-            <div className="prose max-w-full md:col-span-3 lg:col-span-2">
+          <div className="gap-4 md:grid md:grid-cols-5 lg:grid-cols-5">
+            <div className="prose max-w-full leading-5 md:col-span-3 lg:col-span-3">
```

Comment:
why `leading-5`? `prose` sets `line-height` by default

Reply:
This is according to @tiagov8's feed back for #118. The default line height of `prose` is too large. I added `leading-5` to override. 

Looks like I didn't add this change in my previous PR.

I'm OK with keeping the larger line height. IMO this smaller one is ugly.

Reply:
I think it should be modified in the tailwind configuration so its consistent

Reply:
Strange. I removed the `leading-5` and ended up with the same line height, despite it being larger in production at the moment. Unsure why that's happened but I suppose it saves me from making more changes.